src/components/OrgFile/components/TaskListModal/index.js

```diff
@@ -72,15 +72,19 @@ class TaskListModal extends PureComponent {
     const date = new Date();
 
     const filterSuggestions = ['test'];
-    const text = filterSuggestions[0];
```

Comment:
I was just reading through your code when you made the force push. Then I pulled your new commits, but the application didn't start anymore. The variables `text` and `i` couldn't be found.

You're likely seeing it yourself, so just a friendly heads up(;

Reply:
Ups, I forgot to remove that line. Fixed.

Reply:
Sweet^^